src/redux/reducer.js

```diff
@@ -348,13 +356,18 @@ export const reducer = createReducer(initialState, {
     },
 });
 
-function synchCurrentTreeNode(state) {
+function synchCurrentTreeNode(state, parentNodeId = undefined) {
```

Comment:
rename to parentNodeUuid

Reply:
Maybe we could even factorize more this function
function synchCurrentTreeNode(state,nextCurrentNodeUuid) {
// check if its in model and get Node objetc
// overwrite state.currentTreeNode

use synchCurrentTreeNode(state, parentNodeUuid) in NETWORK_MODIFICATION_TREE_NODES_REMOVED
if currentTreeNode is included in nodes deleted
otherwise
use synchCurrentTreeNode(state, undefined)

But ok will see later

Reply:
Ok, done.